src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1680,6 +1685,20 @@ namespace winrt::TerminalApp::implementation
         return nullptr;
     }
 
+    // Method Description:
+    // - returns a tab corresponding to a view item. This might return null,
+    //   so make sure to check the result!
+    winrt::TerminalApp::TabBase TerminalPage::_GetTabByViewItem(const Microsoft::UI::Xaml::Controls::TabViewItem& tabViewItem) const noexcept
```

Comment:
nit:

```suggestion
    winrt::TerminalApp::TabBase TerminalPage::_GetTabByTabViewItem(const Microsoft::UI::Xaml::Controls::TabViewItem& tabViewItem) const noexcept
```

(since ViewItem isn't a thing!)